CIP-0124/README.md

```diff
@@ -0,0 +1,222 @@
+  - Aaron Schmid <aaron@entropilabs.io>
+  - Carolina Isler @LaPetiteADA <lapetiteada@granadapool.com>
+Implementors: []
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/488
+Created: 2023-03-19
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines an additional property to the metadata standard for tokens (NFTs and FTs) to support text localization.
+
+## Motivation: why is this CIP necessary?
+
+Current token metadata only supports a single hardcoded language (mostly English), which limits the accessibility to a certain culture. To get closer to mass adoption, we need to bring down language barriers by extending the current standard to support translations. This is especially relevant for games, metaverse solutions, and RealFi use cases of NFTs.
+
+## Specification
+
+This proposal follows the same specifications as [CIP-0025](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0025) (all versions).
+
+The name of a culture consists of its [[ISO-639]](https://www.iso.org/standard/4767.html) language code with small letters and its [[ISO-3166]](https://www.iso.org/standard/63545.html) country/region code with capital letter separated by a dash "-". For instance, this proposal was written in "en-US": English with the US culture.
+
+This convention is compatible with most operative systems (Linux and Windows) and widely used translation software.
+
+### General structure
+
+The extended JSON metadata standard (CIP-25) allows flexible off- and on-chain string tranlations:
+
+```
+{
+  "721": {
+      "<policy_id>": {
+        "<asset_name>": {
+          "name": <string>,
+          "image": <uri | array>,
+          "mediaType": image/<mime_sub_type>,
+          "description": <string | array>,
+          "files": [{
+            "name": <string>,
+            "mediaType": <mime_type>,
+            "src": "<uri | array>"
+          }],
+
+          <other properties>
+
+          "strings": {
+              "de-CH": {
+                "name": <string in Swiss German>,
+                "image": <localized uri for Swiss German | array>,
+                "description": <string in Swiss German | array>
+                 <other localized properties>
+              },
+              "it-IT": {
+                "name": <string in Italian>,
+                "image": <localized uri for Italian | array>,
+                "description": <string in Italian | array>
+                <other localized properties>
+              },
+
+              <other languages and cultures>
+          }
+        },
+      },
+      "version": <version_id>,
+
+      <information about collection>
+
+      "strings": {
+              "de-CH": {
+                 <localized information about collection in de-CH>
+              },
+              "it-IT": {
+                 <localized information about collection in it-IT>
+              },
+
+              <other languages and cultures>
+      }
+  }
+}
+```
+
+### References
```

Comment:
```suggestion
## References
```
Please move this section to the end by the convention that evolved during this PR's lifetime: at H2 level (`##`) and before the Copyright.  (see CIP-0001 > optional sections)

Reply:
I followed the structure of [CIP-25](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0025), but I can move this section as requested.

Reply:
Done! ;-)

Reply:
@granada-pool I believe it should also be changed in CIP-0025.  At the time I re-reviewed CIP25 at the beginning of the year, it seemed its Reference applied directly to the spec above it... now I can see they're more applicable to the document as a whole.  Thanks for pointing that out & we'll attend to the CIP25 structure at the next opportunity.